src/cli/cli.cpp

```diff
@@ -7573,6 +7589,23 @@ template <> otError Interpreter::Process<Cmd("timeinqueue")>(Arg aArgs[])
 
 template <> otError Interpreter::Process<Cmd("dataset")>(Arg aArgs[]) { return mDataset.Process(aArgs); }
 
+/**
+ * @cli txpower(get,set)
+ * @code
+ * txpower
+ * -10 dBm
+ * Done
+ * @endcode
+ * @code
+ * txpower -10
+ * Done
+ * @endcode
+ * @cparam txpower [@ca{txpower}]
+ * Use the optional `txpower` argument to set the transmit power in dBm.
+ * @par api_copy
```

Comment:
For this one, since it both gets and sets, let's not use the api_copy but instead add a custom description, then do `@sa` for both APIs.